R/create.R

```diff
@@ -32,14 +40,14 @@ quarto_create_project <- function(name, type = "default", dir = ".", no_prompt =
       "This will create a new Quarto {.emph {type}} project as a folder named {.strong {name}} in {.path {xfun::normalize_path(dir)}}."
     ))
     prompt_value <- tolower(readline(sprintf("Do you want to proceed (Y/n)? ")))
-    if (!prompt_value %in% "y") {
+    if (!prompt_value %in% c("", "y")) {
```

Comment:
When I see `(Y/n)`, I think it means that the default (just hitting "enter") is "Y" for "yes". I'm pretty certain that's how it behaves in other R packages that present such a choice. As it stands, if you just hit "enter", you get an error.

Best reference I can find for this: https://askubuntu.com/a/322601